doc/user/radial_and_axial_expansion.rst

```diff
@@ -60,27 +57,235 @@ length, :math:`L_0`,
+  block independently. 
+#. Align blocks axially such that common components have consistent alignments (e.g.,
+  overlapping radial dimensions).
+#. Assign the block lower and upper elevations to account for the thermal expansion of blocks
+  below each block.
+
+   #.  Create new mesh lines that track the target component.
+
+#. Adjust the "Dummy" block located at the top of the assembly to maintain a consistent
+  core-wide assembly height before and after axial thermal expansion is applied.
+
+
+Target Component Logic
+----------------------
+When two or more solid components exist within a Block, the overall height change of the Block is
+driven by an "axial expansion target component" (e.g., fuel). This Component may either be inferred
+from the flags prescribed in the blueprints or manually set using the ``axial expansion target
+component`` block blueprint attribute. The following logic is used to infer the target component:
+
+#. Search Component flags for neutronically important components. These are defined in
+   :py:data:`expansionData.TARGET_FLAGS_IN_PREFERRED_ORDER`.
+#. Compare the Block and Component flags. If a Block and Component contain the same flags, that
+   Component is selected as the axial expansion target Component.
+#. If a Block has :py:data:`flags.flags.PLENUM` or :py:data:`flags.flags.ACLP`, the
+   :py:data:`flags.flags.CLAD` Component is hard-coded to be the axial expansion target component.
+   If one does not exist, an error is raised.
+#. "Dummy Blocks" are intended to only contain fluid (generally coolant fluid), and do not contain
+   solid components, and therefore do not have an axial expansion target component.
+
+
+Mass Conservation
+-----------------
+Due to the requirement that all components within a Block be the same height, the conservation of
+mass post-axial expansion is not trivial. At the Block-level, the axial expansion target component
+is guaranteed to have its mass conserved post-axial expansion. For pinned-blocks, this is typically
+chosen to be the most neutronically important component; e.g., in a fuel Block this is typically the
+Fuel component. All other components, assuming they expand at a different rate than the fuel, will
+exhibit non-conservation on the Block-level as mass is redistributed across the axially-neighboring
+blocks. However, the mass of all solid components at the assembly-level are designed to be conserved
+if the following are met for a given assembly design.
+
+#. Axial continuity of like-objects. E.g., pins, clad, etc.
+#. Components that may expand at different rates axially terminate in unique blocks
+
+   #. E.g., the clad extends above the termination of the fuel and the radial duct encasing an
+      assembly extends past the termination of the clad.
+
+#. The top-most Block must be a "dummy Block" containing fluid (typically coolant).
+
+See :py:module:`armi.tests.detailedAxialExansion` for an example blueprint which satisfy the above
```

Comment:
```suggestion
See :py:module:`armi.tests.detailedAxialExpansion` for an example blueprint which satisfy the above
```
otherwise, broken link
![image](https://github.com/user-attachments/assets/f167dd56-c555-4739-a360-545b147d575b)